deploy/rbac-clowdapp.yml

```diff
@@ -434,6 +434,25 @@ objects:
             value: ${CLOWDER_ENABLED}
           - name: CW_NULL_WORKAROUND
             value: ${CW_NULL_WORKAROUND}
+            #Relations API variables
+          - name: RBAC_CLIENT_ID
```

Comment:
Those variable need to be defined also at bottom in this file - please check others vars - there can be put particular value and here it will just referenced by `${RBAC_CLIENT_ID}`

this way allow to override it for different environment like stage/production and ephemeral - we can just state it in app-interface.